### PR TITLE
[Release Notes] Issue #64927: Constrain implicit raw pointer

### PR DESCRIPTION
**Pull request by atrick**
conversion to bitwise-copyable values

### CODE REVIEW

**Comment by atrick** [comment]
@swift-ci smoke test

**Comment by atrick** [comment]
@tbkka is this ok for release notes?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,30 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
...
+
+  ```swift
+  func inspectData(data: inout Data) {
+    readBytes(&data)
+    // warning: forming an 'UnsafeRawPointer' to a variable of type 'T';
+    // this is likely incorrect because 'T' may contain an object reference.
+  }
+  ```
+
+  Please see the "Workarounds for common cases" section link in this github issue.
```
Comment (tbkka) [inline CHANGELOG.md:29]: I would be explicit here:
```
... in GitHub issue #64927.
```

**Comment by atrick** [comment]
@swift-ci smoke test

**Pull request merged**